<launch>
    <arg name="namespace" default="yk_creator"/>
    <arg name="verbose" default="true"/>
    <group ns="$(arg namespace)">
        <node pkg="vibro_tactile_toolbox" name="reset_joints" type="reset_joints.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="verbose" value="$(arg verbose)"/>
        </node>
    </group>
</launch>